pkgs/applications/graphics/graphui/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, vala
+, pkg-config
+, pantheon
+, python3
+, desktop-file-utils
+, glib
+, graphviz
+, gtk3
+, gtksourceview
+, libgee
+, substituteAll
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "graphui";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "artemanufrij";
+    repo = pname;
+    rev = version;
+    sha256 = "19r44rw0chvj7yrqf35rvpg1yziv25d43h392qnwbmifnq1kbfcs";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
+      graphviz = "${graphviz}/bin/";
+    })
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    vala
+    pkg-config
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    desktop-file-utils
+    glib
```

Comment:
it doesn't seem they include this as a `dependency` object in meson. Will PR that.

Reply:
https://github.com/artemanufrij/graphui/pull/17